doc/sphinx/addendum/extraction.rst

```diff
@@ -99,10 +99,19 @@ Extraction Options
 Setting the target language
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd:: Extraction Language {| OCaml | Haskell | Scheme | JSON }
+.. cmd:: Extraction Language @language
    :name: Extraction Language
 
-   The ability to fix target language is the first and more important
+   .. insertprodn language language
+
+   .. prodn::
+      language ::= OCaml
+      | Ocaml
```

Comment:
Does that contain both `OCaml` and `ocaml`?

Reply:
The text is correct.  It's "OCaml" or "Ocaml".  Not "ocaml" (or "O'Caml").

Reply:
Ok, I just feared this could have been an unintended duplication, sorry for the noise.

Reply:
It's not unintended. The `Ocaml` form is the old form and it is deprecated. I think it has been deprecated for long enough so I suggest:
1. to remove it now
2. to splice `language` so that this part of the doc does not need to be changed.

Reply:
In this PR or a separate one?  If we backport these doc updates but not the removal should we tweak the doc if and when it's backported?

Reply:
If we remove `Ocaml` in `master` (as a separate PR), it's fine to not document it (since it was already not documented) even if this PR is backported IMHO.

Reply:
Otherwise, if we really want to document it, then we should mention that it is deprecated.

Reply:
It has been deprecated in Coq 8.8 (#6261). So I believe we can remove it now.